ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/StatementExecutorTest.java

```diff
@@ -98,7 +145,7 @@ public void shouldHandleCorrectDDLStatement() {
     final CommandId commandId =  new CommandId(CommandId.Type.TOPIC,
                                          "_CorrectTopicGen",
                                          CommandId.Action.CREATE);
-    statementExecutor.handleStatement(command, commandId, Optional.empty());
+    handleStatement(statementExecutor, command, commandId, Optional.empty());
```

Comment:
suggestion: this method _always_ takes `statementExecutor`, so consider making `statementExecutor` a field, if its not already, and drop it from the parameter list.